.github/workflows/request-reviewer.yml

```diff
@@ -24,6 +24,14 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.2
+      - name: Checkout membership data from main branch
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.2
+        with:
+          ref: main
+          path: .
```

Comment:
I *think* this should pull the new file from main and check it out over the old copy from whatever branch the PR was created with, but I'm not sure if `main` will always be GoogleCloudPlatform/main or potentially the fork's main.

Reply:
Can we confirm this refers to GoogleCloudPlatform/main? Otherwise, I'd assume it will fail for most of the PRs.

Reply:
It will have the same `repository` as the previous step, which should be whichever repo the PR is opened on. [See Doc](https://github.com/actions/checkout?tab=readme-ov-file#usage)